flink-tests/src/test/java/org/apache/flink/test/cancelling/JoinCancelingITCase.java

```diff
@@ -30,19 +30,18 @@
 import org.apache.flink.test.util.InfiniteIntegerTupleInputFormat;
 import org.apache.flink.test.util.UniformIntTupleGeneratorInputFormat;
 
+import org.junit.Ignore;
+import org.junit.Test;
+
 /**
  * Test job cancellation from within a JoinFunction.
  */
+@Ignore("Takes too long.")
```

Comment:
Do you know what's taking so long?

Reply:
No, I also didn't check how long it actually takes.